test/unit/amp/amp-story-page-handler.js

```diff
@@ -263,9 +256,70 @@ describe("ampStoryPageHandler unit tests", function () {
       ...dummyOpts,
     });
     assert.strictEqual(getAdditionalConfigCalled, true);
-    assert.strictEqual(additionalConfigReceivedByAmplib, JSON.stringify({
-      key111: 'this contains bk config by default',
-      key222: 'this is the fetched additional config'
-    }))
+    assert.strictEqual(
+      additionalConfigReceivedByAmplib,
+      JSON.stringify({
+        key111: "this contains bk config by default",
+        key222: "this is the fetched additional config",
+      })
+    );
+  });
+  it("Should render timezone if it's passed to seo if timezone is present in publisher config", async function () {
```

Comment:
can change this to 
`should call seo lib with appropriate timezone if timezone info is present in publisher config, pass it on to amplib`